src/coreos-assembler

```diff
@@ -73,7 +73,12 @@ if [ -z "${cmd}" ]; then
 fi
 shift
 
-export COSA_META_SCHEMA="/usr/lib/coreos-assembler/schema/v1.json"
+export COSA_META_SCHEMA="${COSA_META_SCHEMA:-/usr/lib/coreos-assembler/schema/v1.json}"
+schema_override="$(readlink -f "${PWD}"/src/config/schema.json)"
```

Comment:
This is throwing off CI. `readlink` will return 1 if the file doesn't exist. I think we want something like:

```
schema_override="${PWD}/src/config/schema.json"
if [ -e "${schema_override}" ]; then
  export COSA_META_SCHEMA=$(realpath "${schema_override}")
fi
```